src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/TaintedDataAnalysisData.cs

```diff
@@ -5,6 +5,8 @@
 
 namespace Analyzer.Utilities.FlowAnalysis.Analysis.TaintedDataAnalysis
 {
+    // TODO: We should delete this type and use the following instead:
+    //      using TaintedDataAnalysisData = DictionaryAnalysisData<AnalysisEntity, TaintedDataAbstractValue>;
```

Comment:
@dotpaul I have moved PredicateAnalysis based code to a separate sub-type `PredicateAnalysisEntityDataFlowOperationVisitor`. Given that tainted analysis does not use predicated data, you should be able to delete `TaintedDataAnalysisData` completely and just use this using.